### PR TITLE
docs(misc): add module-federeation concepts section

### PR DESCRIPTION
**Pull request by Coly010**
## Current Behavior

Module Federation concept docs are lost in the More Concepts section

## Expected Behavior

Bring Module Federation concepts into their own section

## Related Issue(s)

Fixes #

### CODE REVIEW

File: docs/shared/packages/webpack/webpack-config-setup.md [hunk 1]
```diff
@@ -5,33 +5,38 @@ description: A guide on how to configure webpack on your Nx workspace, and instr
...
+//...
+"options": {
+//...
+"webpackConfig": "apps/my-app/webpack.config.js"
+},
+"configurations": {
+...
+}
+},
+}
```
Comment (isaacplmann) [inline docs/shared/packages/webpack/webpack-config-setup.md:27]: the spacing here doesn't look right

File: docs/shared/packages/webpack/webpack-config-setup.md [hunk 2]
```diff
@@ -40,25 +45,26 @@ Set `isolatedConfig` to `true` in your `project.json` file in the `build` target
...
+"options": {
+//...
+"webpackConfig": "apps/my-app/webpack.config.js",
+"isolatedConfig": true
+},
+"configurations": {
+...
+}
+},
+}
```
Comment (isaacplmann) [inline docs/shared/packages/webpack/webpack-config-setup.md:62]: Spacing got messed up

File: docs/shared/reference/sitemap.md [hunk 3]
```diff
@@ -65,8 +68,6 @@
       - [Nx and the Angular CLI](/concepts/more-concepts/nx-and-angular)
       - [Folder Structure](/concepts/more-concepts/folder-structure)
       - [Nx Devkit and Angular Devkit](/concepts/more-concepts/nx-devkit-angular-devkit)
-      - [Micro Frontend Architecture](/concepts/more-concepts/micro-frontend-architecture)
-      - [Faster Builds with Module Federation](/concepts/more-concepts/faster-builds-with-module-federation)
```
Comment (isaacplmann) [inline docs/shared/reference/sitemap.md:69]: Need to create redirect rules in `nx-dev/nx-dev/redirect-rules.js`

**Review by isaacplmann — CHANGES_REQUESTED** [review]

File: docs/shared/guides/module-federation/micro-frontend-architecture.md [hunk 4]
```diff
@@ -1,9 +1,13 @@
 # Micro Frontend Architecture
 
-Since version 14, Nx provides out-of-the-box [Module Federation](/concepts/more-concepts/faster-builds-with-module-federation) support to both React
-and Angular. The Micro Frontend (MFE) architecture builds on top of Module Federation by providing _independent deployability_.
-
-If you have not read the [Module Federation guide](/concepts/more-concepts/faster-builds-with-module-federation) yet, we recommend that you read it
+Since version 14, Nx provides
```
Comment (ndcunningham) [inline docs/shared/guides/module-federation/micro-frontend-architecture.md:3]: Content seems the same is this a formatting issue?

File: docs/shared/guides/module-federation/micro-frontend-architecture.md [hunk 5]
```diff
@@ -17,9 +21,10 @@ of MFEs and decide whether it makes sense for your own teams.
   logic that breaks compatibility with remotes.
 
 If you are looking at optimizing builds and do not need independent deployments, we recommend reading our guide on
-[Faster Builds with Module Federation](/concepts/more-concepts/faster-builds-with-module-federation).
+[Faster Builds with Module Federation](/concepts/module-federation/faster-builds-with-module-federation).
 
-If you need to use MFEs, keep reading, and we'll examine the architecture and strategies to deal with shared libraries and
+If you need to use MFEs, keep reading, and we'll examine the architecture and strategies to deal with shared libraries
```
Comment (ndcunningham) [inline docs/shared/guides/module-federation/micro-frontend-architecture.md:26]: same formatting issue here

**Review by ndcunningham — COMMENTED** [review]
I think with the formatting and the redirect rules this should be good to go!

File: docs/shared/guides/module-federation/micro-frontend-architecture.md (hunk 4 above)
Comment (Coly010) [reply docs/shared/guides/module-federation/micro-frontend-architecture.md:3]: Yeah, my ide ran code formatting over the md file.

**Review by jaysoo — APPROVED** [review]
Approved, but it looks like there's a setting in your IDE that's auto-formatting markdown files?

The lines that are over limit are being wrapped, but the content remains the same. It's fine since the rendered page remains the same, but maybe check your settings?

**Review by isaacplmann — CHANGES_REQUESTED** [review]
One last formatting change

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**